ProductService/src/main/resources/application.yaml

```
@@ -24,4 +24,15 @@ app:
+    optimization: false
+
+exchange-rate:
+  file-name: exchange-rate.json
+  base-rate: 100.0
+
+currency-service:
+  host: ${CURRENCY_SERVICE_HOST:http://localhost:8080}
+  retry: 3
+  timeout: 1000
+  methods:
+    get-currency: /api/v1/currency
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)